thirdparty/README.md

```diff
@@ -281,23 +281,31 @@ Files extracted from upstream source:
 - `AUTHORS`, `COPYING`, `PATENTS`
 
 
+## mbedtls
+
```

Comment:
You can maybe amend this one to remove the duplicate, so you don't need to also cherry-pick #111079.